packages/core/src/customizations/components/overrides/Navbar.tsx

```diff
@@ -0,0 +1,28 @@
+// This is an example of how it can be used on the starter.
+
+import CustomComponent from '../CustomComponent'
+
+import { SectionOverride } from 'src/typings/overrides'
+
+const SECTION = 'Navbar' as const
+
+const overrides: SectionOverride[typeof SECTION] = {
+  name: SECTION,
+  components: {
+    // Navbar: CustomComponent,
```

Comment:
It's commented  just for test purposes -> See `How to test it?` [section](https://github.com/vtex/faststore/pull/1768)

_uncomment one of these lines and check if your component is being replaced._